src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -394,7 +394,9 @@ final class OnboardingInfoViewController: UIViewController {
 			persistTimestamp(completion: completion)
 		case .enableLoggingOfContactsPage:
 			func handleBluetooth(completion: @escaping () -> Void) {
-				if let alertController = self.exposureManager.alertForBluetoothOff(completion: { completion() }) {
+				if let alertController = self.exposureManager.alertForBluetoothOff(completion: {(
+					
+				)}) {
 					self.present(alertController, animated: true)
 				}
 				completion()
```

Comment:
I'm not quite sure, but as far as I can test, without a device at the moment, the call inside the closure seems to be correct. It's a special case in Debug build settings.
I guess removing line 400 402 should solve the problem.

Reply:
but if I remove 400 , 402 , what is usage of handleBluetooth(completion:) ?
@KaiTeuber